examples/jiralert.yml

```
@@ -40,6 +40,8 @@ receivers:
   - name: 'jira-ab'
     # JIRA project to create the issue in. Required.
     project: AB
+    # Define the jira field used by jiralert to avoid ticket duplication. Optional (default: Labels)
+    field_labels: Labels
```

Hm, what about `id_field`? 

```suggestion
    # The jira field used by jiralert for the unique hash and optional grouping and static labels that identify the ticket for reuse. It's recommended to keep using labels field, but some users prefer custom fields (see issue #164). Optional (default: labels).
    id_field: labels
```